build.sbt

```diff
@@ -14,14 +13,12 @@ lazy val commonSettings = List(
   Compile / console / scalacOptions --= Seq("-deprecation", "-Xfatal-warnings", "-Xlint")
 )
 
-// Cross-compilation does not work for 2.11, haven't found the right combination of (sbt-protoc and sbt) versions
```

Comment:
can we build the codegen for 2.13?

Reply:
SBT is on 2.12, there is no point in compiling for 2.13 (yet).